packages/browser/src/core/analytics/index.ts

```diff
@@ -503,58 +516,20 @@ export class Analytics extends Emitter {
     return integrations
   }
 
-  // analytics-classic stubs
-
-  log() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  addIntegrationMiddleware() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  listeners() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  addEventListener() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  removeAllListeners() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  removeListener() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  removeEventListener() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  hasListeners() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  // This function is only used to add GA and Appcue, but these are already being added to Integrations by AJSN
-  addIntegration() {
-    console.warn(deprecationWarning)
-    return
-  }
-
-  add() {
+  /* analytics-classic stubs */
+  stub(this: never) {
```

Comment:
Does this need to be a public method on the class? Seems like this can be a standalone function that we assign to the deprecated methods.